### PR TITLE
Update minimum wage rates

### PR DESCRIPTION
**Pull request by suzannehamilton**
Update worker and employer minimum wage calculators with the new minimum wage and living wage rates.

Also use the 2018-2019 rates in the regression tests for these calculators.

https://trello.com/c/P1a8k7jl/122-21-april-minimum-wage-calculator-for-workers
https://trello.com/c/2dDKFQ55/123-2-1-april-minimum-wage-calculator-for-employers

https://docs.google.com/document/d/1Ik5E2r4ixEq_hPm_f9wT4LjwZQbfxY3uvtCXCipguZ8/edit

We had to remove the 2009-2012 options from the "past payment" question in both minimum wage smart answers.  This allowed us to remove an outcome about apprentice minimum wage not being available prior to 2010.  We _may_ add new logic in the future to re-instate this and/or combine it with logic to heck the "You can go back up to 6 years" statement on the same question.

### CODE REVIEW

**Comment by suzannehamilton** [comment]
@h-lame Sorry, forgot to update this PR. I've talked to Mark, and he's confirmed that we don't need to do anything with the change to the weekly rate. 👍

File: test/integration/smart_answer_flows/am_i_getting_minimum_wage_test.rb [hunk 1]
```diff
@@ -343,7 +343,7 @@ class AmIGettingMinimumWageTest < ActiveSupport::TestCase
           setup do
             Timecop.travel('07 April 2016')
           end
-          context 'when he is paid over the minimum/living wage' do
+          context 'when they are paid over the minimum/living wage' do
```
Comment (suzannehamilton) [inline test/integration/smart_answer_flows/am_i_getting_minimum_wage_test.rb]: 👍

File: test/integration/smart_answer_flows/am_i_getting_minimum_wage_test.rb [hunk 2]
```diff
@@ -658,11 +914,11 @@ class AmIGettingMinimumWageTest < ActiveSupport::TestCase
...
 
       assert_current_node :past_payment_below
-      assert_equal 6.08, current_state.calculator.minimum_hourly_rate # rate on '2011-10-01'
+      assert_equal 6.19, current_state.calculator.minimum_hourly_rate # rate on '2012-10-01'
 
-      expected_underpayment = 46.18
+      expected_underpayment = 49.98
       # (hours worked * hourly rate back then - paid by employer) / minimum hourly rate back then * minimum hourly rate today
-      # (40h * £6.08 - £200.0) / 6.08 * 6.50 = 46.18
+      # (40h * £6.08 - £200.0) / 6.19 * 6.50 = 46.18
```
Comment (elenatanasoiu) [inline test/integration/smart_answer_flows/am_i_getting_minimum_wage_test.rb]: Shouldn't this be `(40h * £6.19 - £200.0) / 6.19 * 6.50 = 49.98` ?

Comment (h-lame) [reply test/integration/smart_answer_flows/am_i_getting_minimum_wage_test.rb]: oops, yes, cursed comments.

File: test/integration/smart_answer_flows/am_i_getting_minimum_wage_test.rb (hunk 1 above)
Comment (elenatanasoiu) [reply test/integration/smart_answer_flows/am_i_getting_minimum_wage_test.rb]: 👍💐

**Pull request merged**